src/main/java/com/redhat/devtools/lsp4ij/features/foldingRange/LSPFoldingRangeBuilder.java

```diff
@@ -194,6 +199,29 @@ protected String getLanguagePlaceholderText(@NotNull ASTNode node, @NotNull Text
         return null;
     }
 
+    private static boolean isCollapsedByDefault(@NotNull FoldingRange foldingRange) {

```

Comment:
I think it would be very nice to provide the capability to customize collapsed folding.

I suggest that you move (as non static) in the `LSPFoldingFeatures#isCollapsedByDefault(@NotNull FoldingRange foldingRange) ` and update also the LSPApi.md